build/distributions/CentOS/7/x86_64/yum_data/packages.yaml

```diff
@@ -27,32 +27,26 @@ chkrootkit:
   :rpm_name: chkrootkit-0.50-4el7.x86_64.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/7/chkrootkit-0.50-4el7.x86_64.rpm/download.rpm
 clamav:
-  :rpm_name: clamav-0.101.2-1.el7.x86_64.rpm
-  :source: http://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-0.101.2-1.el7.x86_64.rpm
+  :rpm_name: clamav-0.102.4-1.el7.x86_64.rpm
+  :source: http://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-0.102.4-1.el7.x86_64.rpm
 clamav-data:
-  :rpm_name: clamav-data-0.101.2-1.el7.noarch.rpm
-  :source: http://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-data-0.101.2-1.el7.noarch.rpm
+  :rpm_name: clamav-data-0.102.4-1.el7.noarch.rpm
+  :source: http://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-data-0.102.4-1.el7.noarch.rpm
 clamav-devel:
-  :rpm_name: clamav-devel-0.101.2-1.el7.x86_64.rpm
-  :source: http://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-devel-0.101.2-1.el7.x86_64.rpm
+  :rpm_name: clamav-devel-0.102.4-1.el7.x86_64.rpm
+  :source: http://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-devel-0.102.4-1.el7.x86_64.rpm
 clamav-filesystem:
-  :rpm_name: clamav-filesystem-0.101.2-1.el7.noarch.rpm
-  :source: http://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-filesystem-0.101.2-1.el7.noarch.rpm
+  :rpm_name: clamav-filesystem-0.102.4-1.el7.noarch.rpm
+  :source: http://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-filesystem-0.102.4-1.el7.noarch.rpm
 clamav-lib:
-  :rpm_name: clamav-lib-0.101.2-1.el7.x86_64.rpm
-  :source: http://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-lib-0.101.2-1.el7.x86_64.rpm
-clamav-scanner-systemd:
-  :rpm_name: clamav-scanner-systemd-0.101.2-1.el7.x86_64.rpm
-  :source: https://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-scanner-systemd-0.101.2-1.el7.x86_64.rpm
-clamav-server-systemd:
-  :rpm_name: clamav-server-systemd-0.101.2-1.el7.x86_64.rpm
-  :source: https://mirror.umd.edu/fedora/epel/7/x86_64/Packages/c/clamav-server-systemd-0.101.2-1.el7.x86_64.rpm
```

Comment:
:+1: Note to self: These packages were removed after clamav [0.101.4-3](https://src.fedoraproject.org/rpms/clamav/blob/master/f/clamav.spec#_711): https://src.fedoraproject.org/rpms/clamav/pull-request/7#request_diff